### ISSUE TITLE
Receiving A Stake Causes Wallet To Look Like Coins Are Missing

### ISSUE THREAD

**Issue by ecacryptonaut** [Electra-project/electra-desktop#120]
We've received numerous complaints/suggestions about this. When the new wallet receives a stake it makes it look like there is ECA missing. In a future update, would it be possible to have a pop-up note or something more detailed to let the user know what's going on?

**Comment by ivangabriele** [comment]
Yes it's a complicated bug to reproduce because I need to have received a reward in order to see (and fix) the bug but I confirm that there is an issue there.

**Comment by Jenova7** [comment]
Just a display issue, the wallet show coins after 11 confirmations (stake+input)
